pre_build/cccp_index_pre_build_check.py

```diff
@@ -26,7 +27,9 @@ def projectify(
     """
     Projectifying container-index entry to be used for generating job template.
 
-    This function puts all the parameter from container-index entry to a dictory named project. this dictionary later used for rendering the job template.
+    This function puts all the parameter from container-index entry to a
+    dictory named project. this dictionary later used for rendering the job
```

Comment:
typo: dictory --> dictionary

typo: named --> namely

typo .this dictory --> .This dictionary